src/openforms/prefill/api/serializers.py

```diff
@@ -12,12 +12,15 @@
 
 
 class PrefillPluginSerializer(PluginBaseSerializer):
-    requires_auth = serializers.CharField(
-        label=_("Required authentication attribute"),
-        help_text=_(
-            "The authentication attribute required for this plugin to lookup remote data."
+    requires_auth = serializers.ListField(
+        child=serializers.CharField(
+            label=_("Required authentication attribute"),
+            help_text=_(
+                "The authentication attribute required for this plugin to lookup remote data."
+            ),
+            allow_null=True,
         ),
-        allow_null=True,
+        default=list,
```

Comment:
I can't imagine a prefill plugin not needing any auth (except for the demo, but we're changing that behaviour now too), so I'd expect this to be `required=True` without a default, and no `allow_null` for each item.